.devcontainer/.dockerignore

```diff
@@ -0,0 +1,34 @@
+# Build directories and binary files
```

Comment:
I am not familiar with .dockerignore. What is its purpose? Why does not it use the gitignore file?

Reply:
At the very end of the docker image creation when we add the workspace we ignore the files listed in that file. Well I don't know the rationale why it is not using the .gitignore file. For dev-containers it seems logical to me but I as discussed [here](https://stackoverflow.com/questions/58707272/should-dockerignore-typically-be-a-superset-of-gitignore) there are many other use cases for docker out there.

Reply:
Is there a way to automate copying `.gitignore` and `Dockerfile` to these .devcontainer files? Not everyone uses VsCode